app/views/teachers/_teacher_info.html.erb

```diff
@@ -0,0 +1,70 @@
+<div class="card">
+  <div class="card-header">
+    <div class="row d-flex">
+      <div class="col-sm-8 d-flex">
+        <h3>
+          <%= teacher.full_name %>
+        </h3>
+      </div>
+      <div class="col-sm-4 text-right">
+        <%= link_to "Edit Information", edit_teacher_path(teacher), :class=>"btn btn-primary" %>
+      </div>
+    </div>
+  </div>
+  <div class="card-body ml-5">
+    <div class="row mt-3 pb-3">
+      <div class="col-sm-4">
+        <h5> Email </h5>
+        <a href = <%= "mailto:#{teacher.email}" %>> <%= teacher.email %> </a>
+      </div>
+
+      <div class="col-sm-4">
+        <h5> Snap<i>!</i> Username </h5>
+        <%= snap_link(teacher) %>
+      </div>
+
+      <div class="col-sm-4">
+        <h5> Application Status </h5>
+        <%= teacher.display_application_status %>
+      </div>
+    </div>
+
+    <div class="row mt-5 pb-3">
+      <div class="col-sm-6">
+        <h5> Personal or Course Website </h5>
+        <% if teacher.personal_website.empty? %>
+          None provided
+        <% else %>
+          <%= teacher.personal_website %>
+        <% end %>
+      </div>
+
+      <div class="col-sm-6">
+        <h5> Education Level Target </h5>
+        <%= teacher.education_level.capitalize %>
+      </div>
+    </div>
+
+    <div class="row mt-5 pb-3">
+      <div class="col-sm-12">
+        <h5> Current Status </h5>
+        <%= teacher.status.capitalize %>
+        <% if teacher.status == "other" && !teacher.more_info.empty? %>
+          : <%= teacher.more_info %>
+        <% end %>
+      </div>
+    </div>
+
+    <div class="row mt-5">
+      <div class="col-sm-6">
+        <h5> School Name </h5>
+        <a href=<%= teacher.school.website %>><%= teacher.school.name %></a>
+      </div>
+
+      <div class="col-sm-6">
+        <h5> School Location </h5>
+        <%= teacher.school.location %>
+      </div>
+    </div>
+  </div>
+</div>
```

Comment:
Code style is complaining about a lack of newline for a few of the files.

Reply:
Thanks for the catch. I fixed it.